core/src/Microsoft.Mcp.Core/Areas/Commands/Runtime/McpRuntime.cs

```diff
@@ -78,21 +77,6 @@ public async ValueTask<CallToolResult> CallToolHandler(RequestContext<CallToolRe
 
         activity?.AddTag(TagName.ToolName, request.Params.Name);
 
-        var subscriptionArgument = request.Params?.Arguments?
```

Comment:
@conniey, what should we do here?   If the Server area is a generic MCP area, should we be injecting Azure related activity context? 

I think we should, but I think we may need a callback / provider / base class so MCP servers (Azure, Fabric, etc) can inject additional code to be run along with each CallToolHandler invocation